config/rbac/namespace.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    control-plane: controller-manager
+  name: system
```

Comment:
csi-addons should gets deployed in csi-addons-system not in system namespace?

Reply:
> csi-addons should gets deployed in csi-addons-system not in system namespace?

`csi-addons-` will get prefixed from kustomize settings.